docs/interaction_code_flow.md

```diff
@@ -589,6 +590,11 @@ Enable or disable widget functionality with the following options.
   Defaults to `true`.
   Shows eye icon to toggle visibility of the user entered password on the Okta Sign-In page. Password is hidden by default, even when this flag is enabled. Passwords are visible for 30 seconds and then hidden automatically.
 
+#### features.showIdentifier
+
+  Defaults to `true`.
```

Comment:
Curious why need this at first place.? Some customers don't like to show identifier at all?

Reply:
@haishengwu-okta with multi-account support on Okta Verify, users that get redirected to SIW to finish signing in may be unsure of which account they are signing in with. Surfacing the identifier will help clear up any confusion for multi-account holders.